include/broker/zeek.hh

```diff
@@ -105,6 +108,22 @@ public:
     return get<std::string>(get<vector>(as_vector()[2])[0]);
   }
 
+  const timestamp ts() const {
+    auto ev_vec = get<vector>(as_vector()[2]);
+    if (ev_vec.size() > 2)
+      return get<timestamp>(ev_vec[2]);
+
+    return timestamp{};
+  }
+
+  timestamp ts() {
+    auto ev_vec = get<vector>(as_vector()[2]);
+    if (ev_vec.size() > 2)
+      return get<timestamp>(ev_vec[2]);
+
+    return timestamp{};
```

Comment:
I wonder whether this should instead return a `std::optional<timestamp>` since it may or may not be present. Is a default-initialized timestamp as meaninful?

Reply:
Makes sense (see comment below). I switched to `std::optional<timestamp>`.